workspaces/marketplace/plugins/marketplace/src/components/MarketplacePluginInstallContent.tsx

```diff
@@ -69,25 +148,179 @@ export const MarketplacePluginInstallContent = ({
     }
   }, [codeEditor, copyToClipboard]);
 
+  const navigate = useNavigate();
+  const theme = useTheme();
+  const borderColor = theme.palette.mode === 'dark' ? '#a3a3a3' : '#c7c7c7';
```

Comment:
Please let us go with the default Card border.

Reply:
Removed the hardcoded color and used another workaround that utilizes width property for now.